fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraFixity.java

```diff
@@ -99,6 +117,13 @@ public RdfStream getDatastreamFixity() {
         }
 
         LOGGER.info("Get fixity for '{}'", externalPath);
+
+        final RdfStream rs = ((FedoraBinary)resource()).getFixity(translator())
```

Comment:
Is this doing the fixity check twice?  Wouldn't it be better to move the createFixityEvent() call into the kernel anyway, so if there was a non-REST API fixity check (e.g., from a sequencer) it would work too?